recipes/oommf/meta.yaml

```diff
@@ -0,0 +1,58 @@
+package:
+  name: oommf
+  version: "1.2b0_20160930b1"  # [unix]
+  version: "1.2b0_20160930b0"  # [win]
+
+# on Windows, we install from precompiled binary packages
+source:  # [unix]
+  git_url: https://github.com/fangohr/oommf  # [unix]
+  git_rev: 1.2b0_20160930b1  # [unix]
+
+build:
+  number: 0
+  skip: True  # [win32]
+
+requirements:
+  build:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+    - curl  # [win]
+    - 7za  # [win]
+  run:
+    - tk 8.5*  # [unix]
+    - tk 8.6*  # [win]
+
+test:  # [unix]
+  commands:  # [unix]
+    - oommf +version  # [unix]
+
+about:
+  home: http://math.nist.gov/oommf/
+  license_file: oommf/LICENSE
+  license: Custom
+  summary: 'Object Oriented MicroMagnetic Framework (OOMMF)'
+
+  description: |
+    The Object Oriented MicroMagnetic Framework (OOMMF) forms a
+    completely functional micromagnetics package. OOMMF is written in
+    C++. For portable user interfaces, it uses Tcl/Tk so that OOMMF
+    operates across a wide range of Unix, Windows, and MacOSX
+    platforms.
+
+    See https://github.com/fangohr/oommf/blob/master/versionlog.txt
+    for interpretation of version numbers.
+
+    This version contains a bulk dmi extension  # [unix]
+    (https://github.com/fangohr/oommf/blob/master/oommf/app/oxs/local/BulkDMI.h)  # [unix]
+    which is not included in the OOMMF binaries and tarballs from NIST  # [unix]
+    (yet).  # [unix]
```

Comment:
I heard rumours they might (i.e. that line selection takes place before the yaml is interpreted). Can we try if this works, and remove it if doesn't?